cirq/protocols/json_serialization_test.py

```diff
@@ -316,6 +316,8 @@ def test_mutually_exclusive_blacklist():
     'TextDiagramDrawer',
     'ThreeQubitDiagonalGate',
     'Timestamp',
+    'TrialResult',
```

Comment:
What happens if we remove this `TrialResult` entry? I don't know where it comes from.